# Bouding box coordinate: top-left and right-bottom points
BoundingBox bounding_box
std_msgs/Int32MultiArray object_mask
---
# Object point cloud
sensor_msgs/PointCloud2 object_pc
